addons/account_full_reconcile/models/account_full_reconcile.py

```diff
@@ -12,21 +12,28 @@ class AccountMoveLine(models.Model):
 
     @api.model
     def compute_full_after_batch_reconcile(self):
-        super(AccountMoveLine, self).compute_full_after_batch_reconcile()
+        aml_id, partial_rec_id = super(AccountMoveLine, self).compute_full_after_batch_reconcile()
         #check if the reconcilation is full
         partial_rec_set = self.env['account.partial.reconcile']
         total_debit = 0
         total_credit = 0
         total_amount_currency = 0
         currency = False
+        # If we have an exchange rate entry, add it's value to amount_currency
+        if aml_id:
+            exchange_rate_aml = self.browse([aml_id])
+            exchange_rate_partial_rec = self.env['account.partial.reconcile'].browse([partial_rec_id])
+            total_amount_currency += exchange_rate_aml.amount_currency
+            self |= exchange_rate_aml
```

Comment:
you're adding exchange_rate_aml in self but redo at line 45
(self + exchange_rate_aml).ids